### DISCUSSION TITLE
Traceback most recent call back

### DISCUSSION THREAD

**Discussion by BinduBalla** [Atri-Labs/atrilabs-engine#606] [Help: Installation & Start]
![atri](https://user-images.githubusercontent.com/88712870/209062745-a86eb6f3-b375-4462-a57e-362cb3bfbbaa.JPG)

![Capture](https://user-images.githubusercontent.com/88712870/209066281-785638a9-83d7-445a-a3fb-9c75eadcf905.JPG)
python version 3.8.8

**Comment by Shubhodeep100** [comment]
I was also getting this 'Traceback' errors on the terminal.
Try restarting your VSCODE and then again run the shell command and then again start Atri. I got this problem resolved which I was getting on my terminal.
